x/ccv/provider/keeper/genesis.go

```diff
@@ -112,12 +110,9 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
 				)
 				return true
 			})
-		} else {
```

Comment:
Why were pending valset changes only stored  if the channelID was not found, previously?

Reply:
Because we were sending them immediately (in every EndBlock).

We would only queue valset changes if the client was not active. Now we always queue regardless of client